src/pages/Trade/trade-styles.ts

```diff
@@ -10,7 +10,22 @@ export const StyledFlex = styled(Flex)`
 
 export const TradeContainer = styled(Flex)`
   @media (min-width: ${theme.breakpoints.xxl}) {
-    padding-left: 60px;
-    padding-right: 28px;
+    padding-left: 16px;
   }
-`
+`;
+
+export const LineSeparator = styled(Flex)`
```

Comment:
it is not used anywhere,  is it?

Reply:
I see that it's being used on PositionsTable in Trade page.

Reply:
yes, it is used on Positions table, but it is taken from positions-table-styles.ts. And on Trade page it is not used

Reply:
oh ok. We can get reed of it then. ty